app/models/domain.rb

```diff
@@ -0,0 +1,12 @@
+class Domain < ApplicationRecord
+  self.table_name = 'domains'
+
+  has_many :email_redirects, class_name: 'Email::Redirect', foreign_key: :domain_id, inverse_of: :domain, dependent: :restrict_with_error
```

Comment:
@dmitry-sinina you can omit `foreign_key` key cause it will be equal to `:domain_id` by default according to Rails convention